vonk/components/components_classes.rst

```diff
@@ -144,6 +144,8 @@ IVonkContext has three major parts, that are explained below:
       IArgumentCollection: Arguments{get;}
 
       IVonkResponse Response {get;}
+
+      string InformationModel {get;}
```

Comment:
Missing IResource.InformationModel earlier in this file

Reply:
Added